pkg/spanconfig/spanconfigtestutils/utils.go

```diff
@@ -32,6 +32,10 @@ import (
 // and "end" keys.
 var spanRe = regexp.MustCompile(`^\[(\w+),\s??(\w+)\)$`)
 
+// systemTargetRe matches strings of the form
+// "{source=(<id>|system),target=(<id>|system)}".
```

Comment:
Simplify the grammar to drop "system". There are already too many "system" in our vocabulary and here used for two different meanings ("system tenant" and "system target"); there's no difference between tenant-id==1 and system, and the test cases render them differently. It would help simplify your parser below. I'd be ok exporting it now too (ParseSystemTarget) like we do for everything else -- we _are_ going to use something like it in spanconfig.store tests shortly anyway.